sem/manager.py

```diff
@@ -305,6 +332,9 @@ def run_simulations(self, param_list, show_progress=True, stop_on_errors=True):
                 can be either a string or a number).
             show_progress (bool): whether or not to show a progress bar with
                 percentage and expected remaining time.
+            log_component (dict, str): Either a python dictionary with the 
+                log_components (to enable) as the key and their respective 
+                log_levels as the value or a string formatted in the NS_LOG variable format. 
```

Comment:
need more info con how log_component (dict) should be formatted. should log_levels (values) be formatted as `'info|warn` or `info,warn`? prefixes, are ignored, right? this should be specified